Models/product.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require("mongoose")
+const Schema = mongoose.Schema
+
+const Product = new Schema({
+    name: String,
+    description: String,
+    ownerId: { type: Schema.Types.ObjectId, ref: 'Owner' },
+    price: { type: Number, default: 0 },
+    category: [String],
```

Comment:
From my lack of knowledge around mongoose schemas, I'm not 100% sure whether `category` should be [String] or declared as an Array. 

`category: {
type: Array,
default: []
}`

or whether it should stay as it is. I am still referring to the mongoose documentation for guidance right now.